server/api/router.go

```diff
@@ -51,6 +51,11 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	router.HandleFunc("/api/v1/store/{id}", storeHandler.Delete).Methods("DELETE")
 	router.Handle("/api/v1/stores", newStoresHandler(svr, rd)).Methods("GET")
 
+	labelsHandler := newLabelsHandler(svr, rd)
+	router.HandleFunc("/api/v1/labels", labelsHandler.Get).Methods("GET")
```

Comment:
now no test for the API of store and label, I will try to do it in one pr through mock some stores.